rust/rsc/src/common/database.rs

```diff
@@ -157,7 +190,226 @@ pub async fn read_dbonly_blob_store(
 // ----------            Update            ----------
 
 // ----------            Delete            ----------
-pub async fn delete_all_jobs(db: &DatabaseConnection) -> Result<DeleteResult, DbErr> {
+pub async fn delete_all_jobs<T: ConnectionTrait>(db: &T) -> Result<DeleteResult, DbErr> {
     tracing::info!("Deleting ALL jobs");
     job::Entity::delete_many().exec(db).await
 }
+
+// --------------------------------------------------
+// ----------         Visible File         ----------
+// --------------------------------------------------
+
+// ----------            Create            ----------
+pub async fn create_many_visible_files<T: ConnectionTrait>(
+    db: &T,
+    visible_files: Vec<visible_file::ActiveModel>,
+) -> Result<(), DbErr> {
+    if visible_files.len() == 0 {
+        return Ok(());
+    }
+
+    let chunked: Vec<Vec<visible_file::ActiveModel>> = visible_files
+        .into_iter()
+        .chunks(MAX_SQLX_PARAMS / 5)
```

Comment:
Should we put a comment at the top of `types` to remind us to update these numbers if those ever change?
Or is there some other mechanism that will force us to remember to update this?
I don't think we want to go through the effort/complexity of creating a macro to calculate this even though thats possible.

Reply:
Yeah, I'll add  a reminder to `types.rs`, we can't put a comment above the actual types since those files are all generated